client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -10,11 +10,23 @@ import { connect } from 'react-redux';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { requestActivityLogs } from 'state/data-getters';
 import ActivityList from '../../components/activity-list';
+import DatePicker from '../../components/date-picker';
 
 class BackupsPage extends Component {
+	state = {
+		currentDateSetting: false,
```

Comment:
Should this be set to today by default?

Reply:
Yes, literally :-) but there's a pretty convoluted chain of events that need to happen for it to get set and for all the respective things to work properly. We can certainly optimize this flow in a subsequent PR if needed.